DigitalEcosystem/utils/misc.py

```diff
@@ -15,6 +15,11 @@
 
 ]
 
+def rmse(y_true, y_pred):
```

Comment:
Spelled out RMSE here. Using `y_true` and `y_pred` was a deliberate choice to keep the interface identical to the other Sklearn metrics (including the naming convention), since this function is just a wrapper around the Sklearn MSE.